frontend/src/app/hierarchy-viewer/importExportHierarchy.dialog.html

```diff
@@ -0,0 +1,39 @@
+<mat-tab-group (selectedIndexChange)="myTabSelectedIndexChange($event)">
+    [selectedIndex]="selectedTab.value">
+    <mat-tab label="Import">
+        <mat-dialog-content class="tab-container">
+            <input type="file" class="input-hidden" (change)="onFileSelected($event)" #fileUpload>
```

Comment:
This could use a bit of css to put the text and button in the same row. A flexbox would probably do the trick. I have some examples throughout the project with it. Let me know if you need help with that.